### PR TITLE
UIB webhook-template documentation

### PR DESCRIPTION
**Pull request by uibdev**
#### Summary
Documentation on setting up UIB Webhook template.

#### Screenshots
uib-template

#### Changes
- Added a new folder uib
- Created _index.md for uib documentation and added a png file
#### Checklist
- [*] Scope: The referenced issue is addressed, there are no unrelated changes.
- [*] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [*] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [*] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [*] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/integrations/cloud-integrations/uib/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: "UIB"
+description: ""
+weight: 
+aliases: ["/integrations/cloud-integrations/uib/tts-setup"]
```
Comment (johanstokking) [inline doc/content/integrations/cloud-integrations/uib/_index.md:5]: There's no alias needed when this page isn't moved. We use aliases to make sure that old links are still working.

File: doc/content/integrations/cloud-integrations/uib/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+[UIB's](https://www.uib.ai/) technology makes human to machine communications simple.
+
+<!--more-->
+
+## Prerequisites
+
+To send a message via WhatsApp, you need to have a WhatsApp Business Account and the content of the notification message which you need to send to the customer has to be submitted to WhatsApp and approved, before it can be used.
+
+Please send us an email to support@uib.ai, with the company details, Facebook Business Manager ID and an approximate no. of messages you want to send per month. Our support team will revert back with the pricing and next steps.
```
Comment (johanstokking) [inline doc/content/integrations/cloud-integrations/uib/_index.md:16]:
```suggestion
Please send UIB an email to [support@uib.ai](mailto:support@uib.ai), with your company details, Facebook Business Manager ID and an approximate no. of messages you want to send per month. UIB's support team will revert back with the pricing and next steps.
```

File: doc/content/integrations/cloud-integrations/uib/_index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Please note that the template/notification name, name space, access key etc will be provided once the notification message content is approved by WhatsApp.
+
+
+## Configure {{% tts %}}
+
+In order for UIB to be able to decode the data coming from {{% tts %}}, you need to create an uplink [payload formatter]({{< ref "/integrations/payload-formatters" >}}) on {{% tts %}} in order to decode the uplink payload and set fields in the `decoded_payload` object of the uplink message. The notification message will be sent to user if the `decoded_payload_warnings` is not empty. You can push a text to the warnings array according to your parameter thresholds(field1,field2 etc can be the data of your device, like temprature or pressure). The example uplink payload formatter is shown below.
```
Comment (johanstokking) [inline doc/content/integrations/cloud-integrations/uib/_index.md:23]:
```suggestion
In order for UIB to be able to decode the data coming from {{% tts %}}, you need to create an uplink [payload formatter]({{< ref "/integrations/payload-formatters" >}}) on {{% tts %}} in order to decode the uplink payload and set fields in the `decoded_payload` object of the uplink message. The notification message will be sent to user if the `decoded_payload_warnings` is not empty. You can push a text to the warnings array according to your parameter thresholds (`field1`, `field2` etc can be the data of your device, like temperature or pressure). The example uplink payload formatter is shown below.
```

**Review by johanstokking — COMMENTED** [review]
Thanks! A few minor suggestions

**Comment by uibdev** [comment]
Hi
We have made the modifications and committed the changes. Can you please
review the pull request(
https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/365).

**Comment by uibdev** [comment]
Hi Nejra,

We have made an email update in the documentation  and committed the
changes. Can you please review the pull request(
https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/367).

**Pull request closed without merging**